auditlog_security/models/auditlog_rule.py

```diff
@@ -41,15 +42,37 @@ def _get_view_log_lines_action(rule):
                rule.auditlog_line_access_rule_ids.mapped('field_ids').ids)
         )
         return  {
-            "name": _("View log lines"),
+            "name": _("View Log Lines"),
             "res_model": "auditlog.log.line",
             "src_model": rule.model_id.model,
             "binding_model_id": rule.model_id.id,
             "domain": domain
         }
```

Comment:
We should either remove this function or, let it return the server action if it exists.